### ISSUE TITLE
Bais in the conv layer

### ISSUE THREAD

**Issue by Limingxing00** [hkchengrex/STCN#70]
Hi,

I find details about resnet 50 and resnet 18. In resnet 50, the conv layers don't have bais. However, in resnet 18, the conv layers have bais. They are all initialized from pre-training. Could you tell me why?

**Comment by hkchengrex** [comment]
I don't think I have specifically made that distinction anywhere.

**Comment by hkchengrex** [comment]
If you don't mean resnet50 vs resnet18 but just bias in general, I have probably added bias to compensate for the frozen batch norm. But that really shouldn't matter. And we did load the model with `strict=False`.

**Comment by Limingxing00** [comment]
Thanks. I agree with you.
